In a nondescript  <b>office park</b> in the northeast side of  <a class="wiki" href="http://en.wikipedia.org/wiki/Menlo_Park,_California" cat="place	city">Menlo Park, CA</a> , the next  <b>genomics</b>  <b>revolution</b> may be taking place. There, 12 prototypes of a new  <b>sequencing machine</b> developed by startup  <a class="wiki" href="http://en.wikipedia.org/wiki/Pacific_Biosciences" cat="company">Pacific Biosciences</a>  are churning out reams of  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA_sequence" cat="">DNA sequence</a>  as fast as built-in cameras can record it. The deep freezer-size boxes, covered for the time being in red  <b>plastic</b> sheeting, are performing a novel feat: reading  <b>single strands</b> of  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA" cat="">DNA</a>  in  <b>real time</b>. The machine's creators hope that innovation will result in a process that operates fast and cheaply enough to make  <b>sequencing</b> a routine component of medical care.<br><br>The company, founded in  <b>2004</b>, made a  <b>splashy</b> debut almost a year ago, showcasing its  <b>technology</b> to the  <b>scientific community</b> for the first time at a  <b>sequencing</b> conference in  <a class="wiki" href="http://en.wikipedia.org/wiki/Florida" cat="colony	state">Florida</a> .  <a class="wiki" href="http://en.wikipedia.org/wiki/Steve_Turner_(guitarist)" cat="guitarist	person	member	musician">Steve Turner</a> , the company's founder and  <b>chief technology officer</b>, wowed the audience with a video of  <b>single molecule sequencing</b> in progress, the product of a  <b>proof-of-principle</b>  <b>experiment</b> reading a 150-base-pair length of  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA" cat="">DNA</a> . Since then, the startup has garnered $100 million more in funding--for a total of $178 million to date.<br><br>Pacific Biosciences' central innovation is a small chip composed of a 100-nanometer-thick  <b>metal</b>  <b>film</b> deposited on a  <b>silicon-dioxide</b>  <b>substrate</b> and dotted with  <b>thousands</b> of tiny wells, each only tens of  <b>nanometers</b> in  <b>diameter</b>. Before sequencing begins, an  <b>enzyme</b> called  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA_polymerase" cat="">DNA polymerase</a>  is immobilized at the bottom of the well, along with the strand of  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA" cat="">DNA</a>  to be read. Fluorescently labeled bases--each of the four  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA" cat="">DNA</a>  letters labeled with a different marker--are then flooded onto the plate, randomly diffusing into each well. When the correct base diffuses into the bottom of a well, the  <b>enzyme</b> attaches it to the growing strand of  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA" cat="">DNA</a> .<br><br>The wells are so small that  <b>fluorescent light</b> shined through the bottom of the plate penetrates only the lower 20 to 30  <b>nanometers</b> of each well, meaning that only the bases being actively attached to the  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA" cat="">DNA molecule</a>  light up. That allows a  <b>camera</b> to capture the  <b>sequencing</b> reaction as it happens, leaving any irrelevant  <b>chemical activity</b> in the dark. "The waveguide is the first  <b>technology</b> to  <b>allow</b> observation of the  <b>polymerase</b> in action at  <b>physiologically</b> relevant concentrations," says Turner.<br><br>In a video showing the  <b>sequencing</b> reactions in action, a series of lights scattered across the screen burst and fade in quick succession. (A computer detects which base is added with each burst by its position within a well.) The machines can currently sequence 12 million bases of  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA" cat="">DNA</a>  per hour, about  <b>one-third</b> of a percent of a  <b>human genome</b>. The video must actually be slowed to be viewed: the reactions occur too fast to be visible to the  <b>human eye</b>.<br><br>Despite its early success, it's not yet clear whether the company's innovative approach will  <b>surpass</b> "next generation" sequencing technologies already in use.  <a class="wiki" href="http://en.wikipedia.org/wiki/Pacific_Biosciences" cat="company">Pacific Biosciences</a>  plans to release a commercial product in  <b>2010</b> and will announce the target  <b>sequencing</b> capacity for that machine early next year. New types of  <b>sequencing</b> machines, such as those developed by  <a class="wiki" href="http://en.wikipedia.org/wiki/Roche_Applied_Science" cat="company">Roche</a>   <a class="wiki" href="http://en.wikipedia.org/wiki/Science" cat="">Applied Sciences</a>  and  <a class="wiki" href="http://en.wikipedia.org/wiki/Illumina_(company)" cat="company">Illumina</a> , have already revolutionized  <b>genomics</b>, allowing scientists to sequence huge volumes of  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA" cat="">DNA</a>  and reportedly dropping the cost of a  <b>human genome</b> to less than  <b>$100,000</b>.<br><br>But Turner is confident that his method has advantages, especially in the clinical  <b>diagnostics</b> market. The most advanced  <b>sequencing</b> approaches currently on the market stop the  <b>sequencing</b> reaction after the addition of each base, wash away  <b>extra bases</b>, snap a picture, and then repeat. Real-time sequencing is faster and uses fewer  <b>chemicals</b>, making it much cheaper. "In the long run, the  <b>reagent</b> cost dominates in  <b>sequencing</b>," says  <a class="wiki" href="http://en.wikipedia.org/wiki/Chad" cat="country	state">Chad</a>   <a class="wiki" href="http://en.wikipedia.org/wiki/Nusbaum" cat="municipality">Nusbaum</a> , codirector of the  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA_sequencing" cat="technique">Genome Sequencing</a>  and Analysis program at the  <a class="wiki" href="http://en.wikipedia.org/wiki/Broad_Institute" cat="laboratory	institution">Broad Institute</a> , in  <a class="wiki" href="http://en.wikipedia.org/wiki/Cambridge,_Massachusetts" cat="town	seats	place">Cambridge, MA</a> .<br><br>Both Roche and  <a class="wiki" href="http://en.wikipedia.org/wiki/Illumina_(company)" cat="company">Illumina</a>  have ramped up speed by running a massive number of sequencing reactions in parallel. However, these methods generate relatively short lengths of sequence, about 50 to 200 bases pairs, which must then be computationally assembled into a  <b>complete sequence</b>. The shorter the piece, the more computationally difficult it is to sew them together. "With a 35 base read, you can't assemble 25 percent of the  <b>genome</b>," says Turner.<br><br> <a class="wiki" href="http://en.wikipedia.org/wiki/Pacific_Biosciences" cat="company">Pacific Biosciences</a>  has already generated what Turner believes to be "the longest  <b>sequencing</b> trace ever made." In a  <b>proof-of-principle</b>  <b>experiment</b>, scientists continuously sequenced the same 135  <b>base circle</b> of DNA 12 times--it was like repeatedly driving around a  <b>rotary</b>. While the company hasn't yet repeated the feat with natural  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA" cat="">DNA</a> , the ability to generate long reads could be especially important when sequencing unknown  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA" cat="">DNA</a>  or stretches with a highly repetitive series of bases. "Many of the dynamic regions of the  <b>genome</b> that are associated with  <b>disease</b> consist of duplicated and complex  <b>repetitive sequences</b> that cannot be accurately assessed at present," says  <a class="wiki" href="http://en.wikipedia.org/wiki/Evan_E._Eichler" cat="">Evan Eichler</a> , a  <b>geneticist</b> at the  <a class="wiki" href="http://en.wikipedia.org/wiki/University_of_Washington" cat="institution	university">University of Washington</a> , in  <a class="wiki" href="http://en.wikipedia.org/wiki/Seattle,_Washington" cat="">Seattle</a> . "Long-sequence reads are necessary to comprehensively understand  <b>human genetic variation</b>."<br><br>Pacific Biosciences' machine may also enable scientists to generate more accurate sequences. Existing methods generate a  <b>consensus sequence</b> by reading the same section of multiple copies of  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA" cat="">DNA</a> , which may have some copying errors, and pooling the results. "All short-read technologies are less accurate than traditional methods," says  <a class="wiki" href="http://en.wikipedia.org/wiki/Nusbaum" cat="municipality">Nusbaum</a> . With  <a class="wiki" href="http://en.wikipedia.org/wiki/Pacific_Biosciences" cat="company">Pacific Biosciences</a>   <b>technology</b>, scientists would theoretically be able to sequence the same piece of  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA" cat="">DNA</a>  multiple times. "That means we'll be able to detect rare  <b>mutations</b> with  <b>unprecedented</b> accuracy, orders of  <b>magnitude</b> better than others," says Turner.<br><br>He adds that the long reads, high accuracy, and quick turnaround time make the  <b>technology</b> ideal for  <b>diagnostics</b>, such as those for  <b>cancer</b>, which involves long stretches of  <b>repetitive sequences</b>, and  <b>infectious disease</b>, in which small sequence changes may drastically change the infectiousness and  <b>virulence</b> of a  <b>pathogen</b>.<br><br>Perhaps the biggest excitement comes from Turner's predictions for the future. He says that it will be easy to further improve the company's  <b>technology</b> with higher-resolution cameras, faster  <a class="wiki" href="http://en.wikipedia.org/wiki/DNA" cat="">DNA</a>  polymerases, and more densely packed  <b>chips</b>. Currently, each chip has  <b>thousands</b> of wells, only about  <b>one-third</b> of which are used. (The remaining two-thirds house either no  <b>enzyme</b>  <b>molecules</b> or more than one, and thus fail to generate useful information.) But the  <b>chips</b>, which are created using  <b>semiconductor fabrication</b> methods, have the capacity to hold 10 million wells.<br><br>Turner calculates that with a  <b>camera</b> that can track one million wells, a  <b>polymerase</b> that operates at about 50 bases per second (the current rate is 10), and full use of all the wells on the plate,  <a class="wiki" href="http://en.wikipedia.org/wiki/Pacific_Biosciences" cat="company">Pacific Biosciences</a>   <b>technology</b> could read 100 gigabases an hour. That translates to full coverage of a human genome--the same genome sequenced 15 times--in just 15 minutes.<br><br>"I'm pretty excited about the possibilities of this  <b>technology</b>, but I remain to be convinced," says  <a class="wiki" href="http://en.wikipedia.org/wiki/Nusbaum" cat="municipality">Nusbaum</a> . "They still have some significant technical hurdles ahead of them<br>
